schema-engine-tag/src/main/java/org/apache/iotdb/db/metadata/tagSchemaRegion/tagIndex/Request/DeletionRequest.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.iotdb.db.metadata.tagSchemaRegion.tagIndex.Request;
+
+import org.apache.iotdb.lsm.context.requestcontext.RequestContext;
+import org.apache.iotdb.lsm.request.IDeletionRequest;
+
+import java.util.List;
+
+/** Represents a deletion request */
+public class DeletionRequest implements IDeletionRequest<String, Integer, RequestStatus> {
+
+  // tags
+  List<String> keys;
+
+  // int32 id
+  int value;
+
+  // deletion result
+  RequestStatus requestStatus;
```

Comment:
The result or response should be hold and maintained in context rather request.

Reply:
We define the delete method of ILSMEngine as:
`<K, V, R> void delete(IDeletionRequest<K, V, R> deletionRequest);`
If you use context to save the operation result or response, is it necessary to rely on a DeletionRequestContext object inside the DeletionRequest, as well as InsertionRequest and QueryRequest？

Or the delete method can be defined as:
`<K, V,R> R delete(IDeletionRequest<K, V> deletionRequest);`
Which do you think is better, or do you have any other ideas?

Reply:
I think the second one is better. The Request shall not reply on the Response. Request and Response are both defined for and be relied on by the interface.

Reply:
ok, i think the same as you

Reply:
I understand that I save the response result in the context because the response needs to be constructed or updated in multi execution stages, and then modify the insert, delete, query interface methods in ILSMEngine to return the response instead of void